README.md

```diff
@@ -273,11 +314,22 @@ The ID Token contains a `vp` element with the Verifiable Credential data.
       "kty":"EC",
       "x":"jf3a6dquclZ4PJ0JMU8RuucG9T1O3hpU_S_79sHQi7VZBD9e2VKXPts9lUjaytBm",
       "y":"38VlVE3kNiMEjklFe4Wo4DqdTKkFbK6QrmZf77lCMN2x9bENZoGF2EYFiBsOsnq0"
+   },
+   "_credential_types":{
+      "https://www.w3.org/2018/credentials/examples/v1/IDCardCredential":[
+         "src1"
+      ]
+   },
+   "_claim_sources":{
+      "src1":{
+         "format":"vp_jwt",
+         "value":"eyJraWQiOiJkaWQ6aW9uOkVpQzZZOV9hRGFDc0lUbFkwNkhJZDRzZUpq...5SRU16ZEdsUWR6SkdTbWNpZlgwIn0.nwxW-8GVL0msMAhZESDZkGC3U00iJgqQXyz3cpfQXIyzqD82A8Eko7nh-7U8-CZ3gl6tdLgxSJEc6nJM7G_-oQ"
```

Comment:
with Aggregated claims, vp_ldp would have to be embedded as a JWT, too?

> Aggregated Claims
JSON object that MUST contain the JWT member whose value is a JWT [JWT] that MUST contain all the Claims in the _claim_names object that references the corresponding _claim_sources member.